tests/__sf_section_docker.test.fish

```diff
@@ -0,0 +1,222 @@
+source $DIRNAME/spacefish_test_setup.fish
+source $DIRNAME/mock.fish
+set -l LOCAL_DOCKER_VERSION 18.06.1
+
+function setup
```

Comment:
I would suggest doing what we've done for most other tests and `cd`ing into `/tmp/tmp-spacefish` before running tests.

Since the tests are creating and deleting files based on the PWD of the user, it may accidentally delete files in the directory they happen to be in. 😅